api/cases/case.js

```diff
@@ -85,6 +85,7 @@ module.exports = (req, res, next) => {
             replaceSectionValues(schema.details, caseData);
         }
         schema.sections.forEach(section => replaceSectionValues(section, caseData));
+        schema.caseId = caseId;
```

Comment:
What's this for?

Reply:
who knows

Reply:
removed because we can grab this  from the router.prarms